package.json

```diff
@@ -10,8 +10,10 @@
     "test": "echo \"Test will be implemented.\" && exit 0",
     "start": "./bin/gtop",
     "lint": "prettier --single-quote --trailing-comma es5 --write 'lib/**/*.js'",
-    "lint-check": "prettier --check --single-quote --trailing-comma es5 'lib/**/*.js'"
+    "lint-check": "prettier --check --single-quote --trailing-comma es5 'lib/**/*.js'",
+    "man": "cd man && marked-man --version 'v1.1.2' --manual 'Gtop Help' --section 1 man.md > gtop.1"
   },
+  "man": ["man/gtop.1"],
```

Comment:
I am not familiar with man in `package.json`. That would add the man entry after `npm i -g gtop`?

Reply:
yes, that's correct

Reply:
Here is an excerpt from the official documentation.
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#man

> **man**
>
> Specify either a single file or an array of filenames to put in place for the man program to find.
> 
> If only a single file is provided, then it's installed such that it is the result from man <pkgname>, regardless of its actual filename.